Cargo.toml

```diff
@@ -524,7 +526,8 @@ reedline = "0.33.0"
 ref-cast = "1.0"
 regex = "1.10.6"
 regress = "0.9.1"
-reqwest = { version = "0.11", default-features = false }
+reqwest = { version = "0.12", default-features = false }
+reqwest11 = { package = "reqwest", version = "0.11", default-features = false }
```

Comment:
Just a sanity check that this will go away when we merge Maghemite back in here?

Reply:
That's right. Need to get everyone updated and pull out the reqwest 0.11 weeds